### PR TITLE
[release] Update release.json and Go modules for 6/7.33.0-rc.7

### PR DESCRIPTION
**Pull request by kacper-murzyn**

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "SECURITY_AGENT_POLICIES_VERSION": "master"
     },
     "release-a6": {
-        "INTEGRATIONS_CORE_VERSION": "7.33.0-rc.5",
+        "INTEGRATIONS_CORE_VERSION": "7.33.0-rc.6",
```
Comment (olivielpeau) [inline release.json:32]: is it expected to use the rc.6 tag of integrations-core even though we're preparing rc.7? (if it's expected, then all good 👍 )

**Review by olivielpeau — APPROVED** [review]
just a question on the integrations-core tag, but other than that all good

**Review by mx-psi — APPROVED** [review]
Looks good once Olivier's comment is addressed

File: release.json (hunk 1 above)
Comment (kacper-murzyn) [reply release.json:32]: Between rc.6 and rc.7 in integrations-core there are no production changes, just CI fixes. I think we missed it somehow during rc6 build (due to the fact that it's not impacting any functionality) and only now this gets picked up. I think this is acceptable.

**Pull request merged**